contrib/metaphysicl/0.2.0/test/testopt_unit.C

```diff
@@ -30,7 +30,7 @@ int main(void)
   NumberArray<3, float> NA = 0;
   std::cos(NA);
 
-  NumberVector<3, float> NV = 0;
```

Comment:
Can we change this to ```(void)NV;``` or something to get rid of the unused variable warning?  Otherwise we're removing test coverage (probably redundant, but I can't swear that) for that constructor.

Reply:
Will do, sorry didn't actually realize this was a constructor for some reason.